src/runtime/runtime_nintendoswitch_svc.go

```diff
@@ -0,0 +1,17 @@
+// +build nintendoswitch
+
+package runtime
+
+import (
+	"unsafe"
+)
+
+// Result nxOutputString(const char *str, u64 size)
+//export nxOutputString
+func NxOutputString(str *byte, size uint64) uint64
```

Comment:
Strings in TinyGo are represented as exactly this, so it might work to simply declare this function as ` func NxOutputString(s string) uint64`. While it is an implementation detail, it is unlikely to change.